src/Main.cc

```diff
@@ -51,6 +51,27 @@ struct PerSocketData {
     string address;
 };
 
+void DeleteSession(int session_id) {
+    Session* session = sessions[session_id];
+    if (session) {
+        auto uuid = session->GetId();
+        auto address = session->GetAddress();
+        session->DisconnectCalled();
+        spdlog::info("Client {} [{}] Disconnected. Remaining sessions: {}", uuid, address, Session::NumberOfSessions());
```

Comment:
This function name and warning doesn't make sense. It's misleading, because you can call `DeleteSession` on timeout, which will be called _before_ manually disconnecting the client.